src/main/java/com/google/cloud/spanner/pgadapter/statements/CopyStatement.java

```diff
@@ -167,7 +173,7 @@ private void queryInformationSchema() throws SQLException {
                 + COLUMN_NAME
                 + ", "
                 + SPANNER_TYPE
-                + " FROM information_schema.columns WHERE table_name = \"?\"");
+                + " FROM information_schema.columns WHERE table_name = ?");
```

Comment:
why remove the quotes?

Reply:
(Sorry for wading in on the comment). This was missed in one of the previous rounds of reviews, so it got merged. As we use the Cloud Spanner JDBC driver internally in PgAdapter, we need to specify query parameters using JDBC-style parameters. The correct format is `?` and not `"?"`.

Reply:
I see. In that case, we need to make sure that setString will put quotes around the table name to preserve casing.

We already made sure that the parser handles casing correctly -- lowercase everything if there are no quotes, preserve casing if there are quotes. If we don't have quotes here the information_schema query will return the wrong results

Reply:
oh yes, you're right. It's a string here, not an identifier. I'll resolve the conversation